extensions/talkDesk/index.ts

```diff
@@ -0,0 +1,18 @@
+import { type Extension } from '@awell-health/extensions-core'
+import { AuthorType, Category } from '@awell-health/extensions-core'
+import { settings } from './settings'
+
+export const TalkDesk: Extension = {
+  key: 'talkDesk',
+  title: 'TalkDesk',
+  icon_url:
+    'https://prd-cdn-talkdesk.talkdesk.com/cdn-assets/latest/talkdesk/brand/main_brand/logo/talkdesk_logo_purple.svg',
```

Comment:
This logo won't work because it's too wide. It's better to use the compact logos for the extensions.

```suggestion
    'https://images.crunchbase.com/c_lpad,f_auto,q_auto:eco,dpr_1/hjtlemfqjidc7i4cck7v',
```